src/main/java/com/epam/ivankaliada/BasicSpringApp/BasicSpringAppApplication.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.ivankaliada.BasicSpringApp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+
+@ComponentScan(basePackages = {"com.epam.ivankaliada.BasicSpringApp"})
```

Comment:
We can skip this annotation. `@SpringBootApplication` already has `@ComponentScan` annotation and uses current package scan by default.

In case you want to specify a specific package to scan, you can also use 
`@SpringBootApplication(scanBasePackages = {"package"})`